NOTICES.txt

```diff
@@ -0,0 +1,28 @@
+# Software License Agreement
+
+**TinyMCE** – [<https://github.com/tinymce/tinymce>](https://github.com/tinymce/tinymce)
+Copyright (c) 2024, Ephox Corporation DBA Tiny Technologies, Inc.
+
+Licensed under the terms of [GNU General Public License Version 2 or later](http://www.gnu.org/licenses/gpl.html).
+
```

Comment:
Notice files are, by convention, related only to third party code. Not the library itself (that's covered by LICENSE.md).
```suggestion
```